tests/console/validate_encrypt.pm

```diff
@@ -37,8 +45,10 @@ sub run {
         verify_cryptsetup_properties($test_data->{cryptsetup}->{device_status}->{properties}, $status->{properties});
     }
     foreach my $dev (sort keys %{$devices}) {
-        verify_restoring_luks_backups(encrypted_device => $devices->{$dev}->{encrypted_device},
-            backup_file_info => $test_data->{backup_file_info}
+        verify_restoring_luks_backups(
+            encrypted_device_path => $devices->{$dev}->{encrypted_device},
+            backup_file_info      => $test_data->{backup_file_info},
+            backup_base_path      => "$test_data->{backup_base_path}_$dev"
```

Comment:
Do we really need this `$dev` postfix? I would really just us value from test data, and in case we need unique names for multiple devices, we should have it per device in test data.

Reply:
The `$dev` comes free without any harm here. First thought was to keep it as i thought that it represents what we are testing and wondering that it would help in debugging in some way, especially if we there was a need for multiple devices. but in second thought i believe that you have a better point essentially, as you answer to my question in the second part of your comment. the per device solution it would simplify the testing in any case. Will fix

Reply:
I remember some problems when more devices that we fixed, not sure if it is related, but worth to run verification with this job that has more disks: https://openqa.suse.de/tests/4576400#step/validate_encrypt/21

Reply:
Good point, makes sense to do VRs for other architectures. But we still should be fine as we have a loop, so will reuse the file in the next iteration.